std/smtp/_helpers.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+/** Based on https://github.com/golang/go/blob/92c732e901a732855f4b813e6676264421eceae9/src/net/smtp/smtp.go
+  * Copyright 2010 The Go Authors. All rights reserved.
+  * Use of this source code is governed by a BSD-style
+  * license that can be found in the LICENSE file.
+  */
+export function validateLine(line: string): void {
```

Comment:
Maybe move into _address_validation.ts to remove the generically named helpers.ts?

Reply:
`_util.ts` would be more consistent. These are private, I wouldn't worry about a generic `_util` module per-directory.